docs/src/main/sphinx/release/release-449.md

```diff
@@ -0,0 +1,72 @@
+# Release 449 (31 May 2024)
+
+## General
+
+* Add support for configuring the query retry policy on the S3 filesystem with
+  the `s3.retry-mode` and `s3.max-error-retries` configuration properties.
+* Add support for adding and dropping fields to records inside an array. ({issue}`22115`)
+* Add an event listener which exposes collected events to an HTTP endpoint. ({issue}`22158`)
+* Fix rare query failure or incorrect results for array types when the data is
+  dictionary encoded. ({issue}`21911`)
+* Fix JMX metrics not exporting for resource groups. ({issue}`21343`)
+
+## BigQuery connector
+
+* Improve performance when listing schemas while the
+  `bigquery.case-insensitive-name-matching` configuration property is enabled. ({issue}`22033`)
+
+## ClickHouse connector
+
+* Add support for pushing down execution of the `count(distinct)`, `corr`,
+  `covar_samp`, and `covar_pop` functions to the underlying database. ({issue}`7100`)
+* Improve performance when pushing down equality predicates on textual types. ({issue}`7100`)
+
+## Delta Lake connector
+
+* Add support for [the `$partitions` system table](delta-lake-partitions-table). ({issue}`18590`)
+* Add support for reading from and writing to tables with
+  [VACUUM Protocol Check](https://github.com/delta-io/delta/blob/master/PROTOCOL.md#vacuum-protocol-check). ({issue}`21398`)
+* Automatically use `varchar` in struct types as a type during table creation
+  when `char` is specified. ({issue}`21511`)
+* Improve performance of writing to Parquet files. ({issue}`22089`)
+* Fix query failure when the `hive.metastore.glue.catalogid` configuration
+  property is set. ({issue}`22048`)
+
+## Hive connector
+
+* Add support for specifying a catalog name in the Thrift metastore with the
+  `hive.metastore.thrift.catalog-name` configuration property. (`10287`)
+* Add support for filesystem caching on the Trino coordinator. ({issue}`21987`)
```

Comment:
What does "on the Trino coordinator" refer to?

Reply:
It seems this one eventually ended up as `Fix failure when filesystem caching is enabled on Trino clusters with a single node`. To clarify, there wasn't any failure, we were just always keeping the filesystem cache disabled on the coordinator for hive connector. This was causing confusion in single node setups, so now we enabled it on coordinator for hive connector.